lib/experimental/Lists/DataList/LinkItem.tsx

```diff
@@ -0,0 +1,40 @@
+import { memo, MouseEventHandler } from "react"
+
+import { Icon } from "@/components/Utilities/Icon"
+import { ItemProps } from "@/experimental/Lists/DataList"
+import { ItemContainer } from "@/experimental/Lists/DataList/ItemContainer"
+import ChevronRight from "@/icons/ChevronRight"
+
+type LinkItemProps = ItemProps & {
+  href: string
+  onClick: MouseEventHandler<HTMLAnchorElement>
+}
+
+export const LinkItem = memo(({ onClick, href, icon, text }: LinkItemProps) => {
+  return (
+    <ItemContainer
+      as="a"
+      text={text}
+      leftIcon={icon}
+      href={href}
+      onClick={(e) => {
+        if (e.ctrlKey || e.metaKey || e.button === 1) {
+          // Let the browser open the link in a new tab, no custom navigation needed
+          return
+        }
+        e.preventDefault()
+        onClick(e)
+      }}
```

Comment:
This shouldn't be necessary when using `Link` from `linkHandler` as the react router's `Link` component already handles this logic.

Reply:
do I get it right that I don't even need the `onClick` handler?